LICENCE

```diff
@@ -75,46 +75,3 @@ scripts/kconfig/olddefconfig.py
 # AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
 # OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-
-Files with 2-Clause BSD licence:
-
-rimage/elf.h
```

Comment:
Why isn't this license copied to rimage repo?

Reply:
There is already a SPDX line in the header. Is that not enough?

Reply:
IANAL, but SPDX is just for source file. LICENSE is for binary distribution and should contains full texts of licenses.
@lgirdwood please comment

Reply:
Probably safest to include the LICENCE file here.I

Reply:
A PR is created to update the LICENSE file in the rimage repo: https://github.com/thesofproject/rimage/pull/4